Mathlib/CategoryTheory/Limits/Preserves/FunctorCategory.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Bhavik Mehta
 
 ! This file was ported from Lean 3 source module category_theory.limits.preserves.functor_category
-! leanprover-community/mathlib commit 7cd8adb7a9d7d0498d2e76c23cd4255f966899f5
+! leanprover-community/mathlib commit 39478763114722f0ec7613cb2f3f7701f9b86c8d
```

Comment:
I've moved this to #3798

Reply:
I've manually reverted the changes to this file in this PR.